app/controllers/decidim/direct_verifications/verification/admin/direct_verifications_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
             @userlist = params[:userlist]
             @workflows = workflows
 
-            processor = UserProcessor.new(current_organization, current_user)
+            processor = UserProcessor.new(current_organization, current_user, session)
```

Comment:
What is this `session` addition for?

Reply:
What I see is that it's passed into `Verification::ConfirmUserAuthorization.call` because its parent, `Decidim::Verifications::ConfirmUserAuthorization` added it as new argument to the constructor in https://github.com/decidim/decidim/pull/5553/files#diff-09b193a1179fc3cd2f2b0c72d929dca8ac2cd7a4dabc468c7a20d140c40faf0a

Reply:
due to this changes https://github.com/decidim/decidim/pull/5553, introduced in decidim@0.22

Reply:
ah ok, thanks